tests/firsttests.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect } from '@playwright/test';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
+    await page.fill('#email', 'customer@practicesoftwaretesting.com');
+    await page.fill('#password', 'welcome01');
+    await page.click('[data-test="login-submit"]')
+    await page.waitForURL('https://practicesoftwaretesting.com/account');
+
+    await expect(page.url()).toBe('https://practicesoftwaretesting.com/account');
```

Comment:
You can remove line 8 and update line 10, by using https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-url

```
await expect(page).toHaveURL('/account');
```